modules/gcp-gke/outputs.tf

```diff
@@ -39,6 +39,6 @@ output "cluster_all_primary_node_pool_tags" {
 }
 
 output "istio_gatekeeper_firewall_rule_self_link" {
-  description = "The self_link attribute of the firewall rule created to allow Gatekeeper and Istio to function."
-  value       = google_compute_firewall.gke_private_cluster_istio_gatekeeper_rules.self_link
+  description = "The self_link attribute of the firewall rule created to allow CP to NP communication"
```

Comment:
This output is not used in the layer models, IMO we should make this a breaking change and drop the output.

Reply:
Assumed it was being used and hence let it be. Dropping this now.

Reply:
Dropped